### ISSUE TITLE
Testing part 1

### ISSUE THREAD

**Issue by jpark134** [sklaw/enee447project6_hw_template_Shuangqi_sessions#1]
I have a bit of confusion I want to clear up in regards to testing our implementation of getting core id.  So, I decided to create a new system call, and I think I'm finished, but I want to make sure I tested it correctly.  What I did was comment out [this](https://github.com/sklaw/enee447project6_hw_template_Shuangqi_sessions/blob/master/z_shell.c#L14) line in z_shell.c, so that run_shell would not just use a constant value for the core id.  I had made an actual get_cpuid() method inside of z_shell that run_shell could call.  From here, after compiling any of the 1_boot files, as long as it shows the proper cpu id for the cores (instead of just showing a constant value of 0), it should be working correctly, right?

Also, trying to do something like RUN CPU 0x[address of get_cpuid() in kernel7.list] would not work because we would have to put the code inside get_cpuid() in an infinite loop, which would mess up the run_shell, so this is not something we should do to test our code, correct?

**Comment by sklaw** [comment]
Yes you could replace the macro `get_cpuid` with an actual function and test it by running the shell. No need to start a thread for testing.
